src/main/java/calculation/Calculator.java

```diff
@@ -0,0 +1,72 @@
+package calculation;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class Calculator {
+    private static final List<String> arithmeticSymbols = Arrays.asList("+", "-", "*", "/");
```

Comment:
정적 상수로 선언해주신 부분 좋네요 👍 
다만 `convention` 규칙에 의해서 대문자로 표기되면 더 좋을 것 같습니다. 

```suggestion
    private static final List<String> ARITHMETIC_SYMBOLS = Arrays.asList("+", "-", "*", "/");
```